src/WorldStorage/WSSAnvil.cpp

```diff
@@ -2665,33 +2665,23 @@ void cWSSAnvil::LoadSilverfishFromNBT(cEntityList & a_Entities, const cParsedNBT
 
 void cWSSAnvil::LoadSkeletonFromNBT(cEntityList & a_Entities, const cParsedNBT & a_NBT, int a_TagIdx)
 {
-	// Wither skeleton is a separate mob in Minecraft 1.11+, but
-	// we need this to load them from older worlds
+	// Wither skeleton is a separate mob in Minecraft 1.11+, but we need this to
+	// load them from older worlds where wither skeletons were only a skeleton with a flag
 	int TypeIdx = a_NBT.FindChildByName(a_TagIdx, "SkeletonType");
 	if (TypeIdx < 0)
 	{
 		return;
 	}
```

Comment:
Is the `SkeletonType` still present in the newer data where witherskeletons are saved as separate entities? If not, this code will drop all skeletons.